CMakeLists.txt

```diff
@@ -19,12 +19,20 @@ if(NOT WIN32)
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+option(DISABLE_SANITIZERS "disables the use of gcc saniztizers")
```

Comment:
Please specify the default value explicitly.

Spelling: saniztizers

Reply:
The specified value is two lines below

Reply:
No, two lines below the value is conditionally changed to `ON`. I want the default value to be specified in this line as the third argument as `OFF`.

Reply:
See 4a128d9.